learn/develop/models/index.qmd

```diff
@@ -47,22 +47,22 @@ This article describes the process of creating a new model function. Before proc
 
 ## An example model
 
-As an example, we'll create a function for _mixture discriminant analysis_. There are [a few packages](http://search.r-project.org/cgi-bin/namazu.cgi?query=%22mixture+discriminant%22&max=100&result=normal&sort=score&idxname=functions) that implement this but we'll focus on `mda::mda`:
+As an example, we'll create a function for _mixture discriminant analysis_. There are [a few packages](http://search.r-project.org/cgi-bin/namazu.cgi?query=%22mixture+discriminant%22&max=100&result=normal&sort=score&idxname=functions) that implement this but we'll focus on `mda::mda()`:
 
 ```{r}
 #| label: "mda-str"
 str(mda::mda)
 ```
 
-The main hyperparameter is the number of subclasses. We'll name our function `discrim_mixture`. 
+The main hyperparameter is the number of subclasses. We'll name our function `discrim_mixture()`. 
 
 ## Aspects of models
 
 Before proceeding, it helps to to review how parsnip categorizes models:
 
 * The model _type_ is related to the structural aspect of the model. For example, the model type `linear_reg` represents linear models (slopes and intercepts) that model a numeric outcome. Other model types in the package are `nearest_neighbor`, `decision_tree`, and so on. 
```

Comment:
do we wanna add `()` to these too since they are function calls.

`linear_reg` -> `linear_reg()`?

Reply:
That was my first impulse as well. I was on the fence if the text was not intentionally distinguishing between the type and the model function but if you read it that way as well, I'll update!